README.md

```diff
@@ -58,17 +58,19 @@ minio:
     tls: true
     annotations:
       cert-manager.io/cluster-issuer: "letsencrypt-staging"
-  secretKey:
-    password: "some-secret-pw"
-  accessKey:
-    password: "some-secret-pw"
+  auth:
+    rootUser: "some-secret-pw"
+    rootPassword: "some-secret-pw"
```

Comment:
Should be documented in release notes in README.md because it will break upgrades without editing the values in existing environments.

Reply:
done